packages/grafana-ui/src/components/Logs/LogRowContextProvider.test.tsx

```diff
@@ -169,7 +169,7 @@ describe('LogRowContextProvider', () => {
         return <></>;
       });
       await act(async () => {
```

Comment:
We should be able to remove this `act`. If there's an async update, maybe wait for something?